.circleci/config.base.yml

```diff
@@ -1303,7 +1303,7 @@ commands:
                   source $BASH_ENV
                   amplify version
                   cd packages/amplify-e2e-tests
-                  retry yarn run e2e --no-cache --maxWorkers=3 --forceExit $TEST_SUITE
+                  retry runE2eTest
```

Comment:
this is the crux of the change. now windows e2e test will use same retry logic as linux.
I.e. when retry happens only tests that failed are re-attempted.